### PR TITLE
[chore] Implement route to retrieve sample entity_serving_names for a deployment

### PR DESCRIPTION
**Pull request by kchua78**
## Description

Implement route to retrieve sample entity_serving_names for a deployment.
This can be used to support preview functionalities for deployments

## Type of Change

Label this pull request to place it under the correct category in Release Notes:

|        **Pull Request Label**         | **Category in Release Notes** |
|:-------------------------------------:|:-----------------------------:|
|       `enhancement`, `feature`        |          🚀 Features          |
| `bug`, `refactoring`, `bugfix`, `fix` |    🔧 Fixes & Refactoring     |
|       `build`, `ci`, `testing`        |    📦 Build System & CI/CD    |
|              `breaking`               |      💥 Breaking Changes      |
|            `documentation`            |       📝 Documentation        |
|            `dependencies`             |    ⬆️ Dependencies updates    |

## Checklist

- [ ] I have read the [`CODE_OF_CONDUCT.md`](https://github.com/featurebyte/featurebyte/blob/main/CODE_OF_CONDUCT.md) and [`CONTRIBUTING.md`](https://github.com/featurebyte/featurebyte/blob/main/CONTRIBUTING.md) guides.
- [ ] I have written tests for the changes made.
- [ ] I have written docstrings in [NumpyDoc format](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard)
- [ ] I have labeled my Pull Request correctly

### CODE REVIEW

File: featurebyte/service/online_serving.py [hunk 1]
```diff
@@ -201,56 +289,20 @@ async def get_request_code_template(  # pylint: disable=too-many-locals
...
-        }
-
-        # populate template
-        with open(
-            file=template_file_path,
-            mode="r",
-            encoding="utf-8",
-        ) as file_object:
-            template = Template(file_object.read())
+                    entities[column.entity_id][
```
Comment (YS-L) [inline featurebyte/service/online_serving.py:293]: If an entity appear in multiple tables, will this repeat the extraction of unique values from those tables?

**Review by YS-L — APPROVED** [review]
LGTM, one comment but could be an existing issue.

File: featurebyte/service/online_serving.py (hunk 1 above)
Comment (kchua78) [reply featurebyte/service/online_serving.py:293]: yes, is there any way to decide which table should be used in such cases?

Comment (YS-L) [reply featurebyte/service/online_serving.py:293]: If the purpose is for getting unique values, I think it makes sense to use the table where the entity is the primary key / natural key if available.

**Pull request merged**